packages/gatsby-admin/package.json

```diff
@@ -15,6 +15,7 @@
     "@emotion/styled": "^10.0.27",
     "@typescript-eslint/eslint-plugin": "^2.28.0",
     "@typescript-eslint/parser": "^2.28.0",
+    "cross-env": "^5.2.1",
```

Comment:
this is a dev dependency :)

Reply:
https://github.com/gatsbyjs/gatsby/pull/24644